src/app/components/MessageBanner/index.styles.tsx

```diff
@@ -0,0 +1,165 @@
+  imageLtr: ({ mq }: Theme) =>
+    css({
+      maxWidth: '184px',
+      [mq.GROUP_3_ONLY]: {
+        maxWidth: '224px',
+        bottom: 0,
+        right: 0,
+        position: 'absolute',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        maxWidth: '224px',
+        bottom: 0,
+        right: 0,
+        position: 'absolute',
+      },
+      img: { objectPosition: 'top' },
+    }),
+  imageRtl: ({ mq }: Theme) =>
+    css({
+      maxWidth: '184px',
+      [mq.GROUP_3_ONLY]: {
+        maxWidth: '224px',
+        bottom: 0,
+        left: 0,
+        position: 'absolute',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        maxWidth: '224px',
+        bottom: 0,
+        left: 0,
+        position: 'absolute',
+      },
+      img: { objectPosition: 'top' },
+    }),
+  linkBackground: ({ mq, palette }: Theme) =>
+    css({
+      padding: '1rem',
+      backgroundColor: palette.WHITE,
+      margin: '0 1rem 1rem 1rem',
+      width: '100%',
+      textDecoration: 'none',
+      '&:hover': {
+        backgroundColor: '#F6F6F6',
+        textDecoration: 'underline',
+        color: palette.BLACK,
+      },
+      '&:focus': {
+        backgroundColor: '#F6F6F6',
+        outline: `${pixelsToRem(3)}rem solid ${palette.WHITE}`,
+        boxShadow: `0 0 0 ${pixelsToRem(3)}rem ${palette.BLACK}`,
+        outlineOffset: `${pixelsToRem(3)}rem`,
+        textDecoration: 'underline',
+        color: palette.BLACK,
+      },
+      [mq.GROUP_3_ONLY]: {
+        width: 'auto',
+        maxWidth: 'calc(100% - 240px)',
+        margin: '0 0 1.5rem 0',
+        paddingBottom: '1rem',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        width: 'auto',
+        maxWidth: 'calc(100% - 240px)',
+        margin: '0 0 1.5rem 0',
+        paddingBottom: '1rem',
+      },
+    }),
+  link: ({ palette }: Theme) =>
+    css({
+      color: palette.BLACK,
+      textDecoration: 'none',
+      '&:hover, &:focus': {
+        textDecoration: 'underline',
+        color: palette.BLACK,
+      },
+      paddingInlineStart: '0.5rem',
+      verticalAlign: 'middle',
+      '&:visited': {
+        color: palette.BLACK,
+      },
+    }),
+  chevron: () =>
+    css({
+      marginInlineStart: '0.5rem',
+      width: '1rem',
+      height: '1rem',
+      verticalAlign: 'middle',
+      fill: 'currentcolor',
+    }),
+  linkAndChevron: () =>
+    css({
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'center',
+    }),
+  flex: ({ mq }: Theme) =>
+    css({
+      display: 'flex',
+      flexDirection: 'column-reverse',
+      alignItems: 'center',
+      [mq.GROUP_3_ONLY]: {
+        display: 'flex',
+        flexDirection: 'row-reverse',
+        justifyContent: 'start',
+        alignItems: 'flex-end',
+        position: 'relative',
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        display: 'flex',
+        flexDirection: 'row-reverse',
+        justifyContent: 'start',
+        alignItems: 'flex-end',
+        position: 'relative',
+      },
```

Comment:
Can these styles be combined under the one `mq.GROUP_3_MIN_WIDTH ` media query?

Reply:
I've sorted all of these and it all works correctly. I also grouped the linear gradients for the two higher breakpoints into the GROUP_3_MIN_WIDTH media query as well to avoid repeating code.

Reply:
Nice! Great work @louisearchibald :)